src/verso/Verso/Code/Highlighted.lean

```diff
@@ -19,6 +19,33 @@ open Std (HashMap)
 
 namespace SubVerso.Highlighting
 
+
+/--
+Serialize `Highlighted` code as a string.
+
+Nothing should be assumed about the `String` output except that it can be passed to `hlFromExport`
+to recover the original input.
+-/
+def hlToExport (hl: SubVerso.Highlighting.Highlighted) : String :=
+  hl.exportCode |>.toJson |>.compress
+
+/--
+Recover the `Highlighted` data from its serialization.
+
+Can only be expected to work on the output of `hlToExport`.
+-/
+def hlFromExport (exportLit : String) : SubVerso.Highlighting.Highlighted :=
```

Comment:
```suggestion
def hlFromExport! (exportLit : String) : SubVerso.Highlighting.Highlighted :=
```
The `!` expresses that it will panic on failure (which should also be mentioned in the API docs)